zha/application/platforms/light/__init__.py

```diff
@@ -1190,6 +1231,12 @@ def available(self) -> bool:
         """Return entity availability."""
         return self._available
 
+    def _persist_color_mode(self, color_mode: ZclColorMode) -> None:
+        """Persist the color mode."""
+
+        # FIXME: Groups use raw clusters, not cluster handlers
```

Comment:
Is "FIXME" needed here?
At least persisting the color mode shouldn't be an issue for groups. When initializing, the LightGroup should `update()` its status by checking all the member statuses which correctly persist/restore the color mode (either being persisted through ZCL cache or if it were to be restored from HA. Although the `__init__` `update()` call might be done a bit too early?).

(I need to find the code for how the group is notified of child updates..)

There's also no attribute cache for groups, since that wouldn't make a ton of sense, so I'm not sure the "raw clusters" vs "cluster handlers" comment makes much sense for this use-case(?) Or am I missing something?

Reply:
The fact that cluster handlers are bypassed is a bug, IMO, since the parent class expects the cluster handler to *be* one, not a cluster instance. We filter most commands through cluster handlers but groups bypass that and wire straight into zigpy. It might be good in the future to create a group cluster handler.